settings.gradle

```diff
@@ -29,7 +29,7 @@ include(":ktruth")
 include(":korge-gradle-plugin")
 include(":korge-reload-agent")
 include(":korge-sandbox")
-if (System.getenv("ENABLE_OSX_METAL_PLAYGROUND") == "true") {
+if (enableMetalPlayground) {
```

Comment:
Alternatively we can use an environment called DISABLE, and updated the `.github/workflows` to disable it, to ensure it is not published to maven central. Or disable publishing in the build gradle

Reply:
Using environment variable is not convenient on IntelliJ, i will switch of the value on the gradle.properties before leaving the draft status.

Reply:
Well, this is for configuring, not for running. But in any case, my main concern here is: not publishing/installing and ideally not building this module in CI. Maybe the quickest solution is `if (System.getenv("CI") != "true") {`. That way on contributor's computers the module would be available directly, and no extra configuration required in workflows.